features/fixtures/mazerunner/cxx-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXPausedSessionScenario.kt

```diff
@@ -0,0 +1,28 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+
+class CXXPausedSessionScenario(
+    config: Configuration,
+    context: Context,
+    eventMetadata: String?
+) : Scenario(config, context, eventMetadata) {
+
+    init {
+        System.loadLibrary("cxx-scenarios")
+        config.autoTrackSessions = false
+    }
+
+    external fun crash(counter: Int): Int
+
+    override fun startScenario() {
+        super.startScenario()
+        if ("non-crashy" != eventMetadata) {
```

Comment:
"non-crashy" should never be mentioned following the previous PR.  It's probably as simple as just removed all the guards though.

Reply:
Removed guards.